### PR TITLE
Bridge add new stake

### PR DESCRIPTION
**Pull request by meho99**

### CODE REVIEW

File: src/hooks/useContractCall.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { FieldValues } from 'react-hook-form';
+import { ContractReceipt, ContractTransaction } from 'ethers';
+
+import { FiniteStates } from '../utils/types';
+
+export const useContractCall = <Values extends FieldValues>(
```
Comment (michalsidzej) [inline src/hooks/useContractCall.ts:7]: Nice

File: src/hooks/useDebounce.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react';
+
+export const useDebounce = <Val>(value: Val, delay: number): Val => {
```
Comment (michalsidzej) [inline src/hooks/useDebounce.ts:3]: Maybe default value? We will probably use the same value 95% of the times.

**Review by michalsidzej — DISMISSED** [review]
GIT

File: src/hooks/useDebounce.ts (hunk 2 above)
Comment (meho99) [reply src/hooks/useDebounce.ts:3]: Agree, will add it

**Pull request merged**